wc-admin.php

```diff
@@ -98,6 +98,10 @@ function deactivate_wc_admin_plugin() {
  * update automatically.
  */
 function wc_admin_init() {
+	if ( ! dependencies_satisfied() ) {
```

Comment:
maybe we should rename this method `wc_admin_dependencies_satisfied` at some point too.